templates/config.edn

```diff
@@ -205,78 +233,86 @@
     :inputs [:today :7d-after]
     :collapsed? false}]}
 
- ;; Add your own commands to slash menu to speedup.
- ;; E.g.
+ ;; Add custom commands to the command palette
+ ;; Example usage:
  ;; :commands
  ;; [
- ;; ["js" "Javascript"]
- ;; ["md" "Markdown"]
- ;; ]
- :commands
- []
-
- ;; By default, a block can only be collapsed if it has some children.
- ;; `:outliner/block-title-collapse-enabled? true` enables a block with a title
- ;; (multiple lines) can be collapsed too. For example:
+ ;;  ["js" "Javascript"]
+ ;;  ["md" "Markdown"]
+ ;  ]
+ :commands []
+
+ ;; Enable collapsing blocks with titles but no children.
+ ;; By default, only blocks with children can be collapsed.
+ ;; Setting `:outliner/block-title-collapse-enabled?` to true allows collapsing
+ ;; blocks with titles (multiple lines) and content. For example:
  ;; - block title
- ;;   block content
+ ;; block content
  :outliner/block-title-collapse-enabled? false
 
  ;; Macros replace texts and will make you more productive.
- ;; For example:
+ ;; Example usage:
  ;; Change the :macros value below to:
  ;; {"poem" "Rose is $1, violet's $2. Life's ordered: Org assists you."}
  ;; input "{{poem red,blue}}"
  ;; becomes
  ;; Rose is red, violet's blue. Life's ordered: Org assists you.
  :macros {}
 
- ;; The default level to be opened for the linked references.
- ;; For example, if we have some example blocks like this:
+ ;; Configure the default expansion level for linked references.
+ ;; For example, consider the following block hierarchy:
  ;; - a [[page]] (level 1)
  ;;   - b        (level 2)
  ;;     - c      (level 3)
  ;;       - d    (level 4)
  ;;
- ;; With the default value of level 2, `b` will be collapsed.
- ;; If we set the level's value to 3, `b` will be opened and `c` will be collapsed.
+ ;; With the default value of level 2, block b will be collapsed.
+ ;; If the level's value is set to 3, block c will be collapsed.
  :ref/default-open-blocks-level 2
 
+ ;; Configure the threshold for linked references before collapsing.
  :ref/linked-references-collapsed-threshold 50
```

Comment:
What number is being counted, and what is collapsed once you hit that number? Unclear.